lib/semiquo.gi

```diff
@@ -51,7 +51,8 @@ function(cong)
     SetOne(Q, One(S)^QuotientSemigroupHomomorphism(Q));
   fi;
 
-  if HasGeneratorsOfSemigroup(S) then
+  if HasGeneratorsOfMagmaWithInverses(S) or
+     HasGeneratorsOfSemigroup(S) then
     Qgens:= List( GeneratorsOfSemigroup( S ),
```

Comment:
Doesn't this assume that if `HasGeneratorsOfMagmaWithInverses` then we can obtain `GeneratorsOfSemigroup`? Is this safe?

Reply:
Probably not. I'll check.

Reply:
Turns out this is a leftover from my experiments and can just go away again.